pkg/elasticsearch/elasticsearch_provider.go

```diff
@@ -94,22 +94,25 @@ func (es ElasticsearchProvider) DeleteService(auth common.IKubernetesAuthInforma
 }
 
 // Converts a v1.Elasticsearch instance to an service representation
-func (es ElasticsearchProvider) CrdInstanceToServiceInstance(crdInstance *v1.Elasticsearch) *service.IService {
+func (es ElasticsearchProvider) CrdInstanceToServiceInstance(auth common.IKubernetesAuthInformation, crdInstance *v1.Elasticsearch, crdApi *kubernetes.CommonCrdApi) *service.IService {
 	yamlData, err := yaml.Marshal(crdInstance)
 	if err != nil {
 		yamlData = []byte("Unknown")
 	}
 
+	mApi, _ := kubernetes.GenerateK8sApiFromToken(es.Host, es.CaPath, auth.GetKubernetesAccessToken())
```

Comment:
Suppressed error could result in an error later on at another place. Method should return an error.